### DISCUSSION TITLE
Telegram适配器无法启用的问题

### DISCUSSION THREAD

**Discussion by NdR60A** [nonebot/discussions#542] [问答] [answered]
![SCR-20241210-uqfp](https://github.com/user-attachments/assets/401a49b1-c0dd-4efd-941c-9a78f5c36c5a)
第一次用这个，然后按照Telegram的适配器的教程一步一步走，然后一启动就这样了，求助，谢谢

**Comment by j1g5awi** [comment, marked as answer]
httpx 上游 break 了。安装最新 commit 的 nonebot2，或者降低 httpx<0.26.0

**Reply by NdR60A** [reply]
感谢回复，我昨天才安装的最新提交的 nonebot2，然而还是一样的无法启动，那么降低 httpx 版本应该如何操作呢，谢谢

**Reply by yanyongyu** [reply]
你安装的是master的nb2？httpx降级使用 `pip install 'httpx<0.28'`

**Reply by NdR60A** [reply]
> 你安装的是master的nb2？httpx降级使用 `pip install 'httpx<0.28'`

阿这，我就是按照教程上安装的，不是很清楚是哪个……，已经降级 hpptx 成功启动了，感谢
